### ISSUE TITLE
Autosuggest: The state of content that can be expanded or collapsed is not announced to screen reader users (WCAG Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2235]
Add `aria-expanded: "false"` to `<input>`...

(Page 27-28 of June 2022 DAC DS report)...

---
## Expandable Content
The state of content that can be expanded or collapsed is not announced to screen reader users or marked-up correctly and may cause disorientation.

### WCAG Reference:
4.1.2 Name, Role, Value (Level A)
Understanding Name, Role, Value | How to Meet Name, Role, Value

### Issue ID: DAC_Expandable_Content_01
URL: https://ons-design-system.netlify.app/
Page title: Home
Journey: Journey 3, Step 2

Content is expanded after the user begins to type characters into the search element of the page. The search element does not contain an aria-expanded attribute with a value of true or false. This means that screen reader users may at first not be aware that content will be expanded.

### Screen reader comments: 
“When testing the ‘install with npnm’ page I found that the search pop up and menu navigation are confusing to VoiceOver users as their states are not announced when changed. I would find it helpful if expanded and collapsed or show/hide can be used to indicate this to VoiceOver users.”

### Solution:
If an element can expend or collapse content, then there must be an aria-expanded attribute with either a value of true or false respectively. This ensures that screen reader users are aware that more content can be displayed.
